CondFormats/BTauObjects/src/BTagEntry.cc

```diff
@@ -17,8 +22,8 @@ BTagEntry::Parameters::Parameters(
   float discr_max
 ):
   operatingPoint(op),
-  measurementType(measurement_type),
-  sysType(sys_type),
+  measurementType(makeLowerCase(measurement_type)),
+  sysType(makeLowerCase(sys_type)),
```

Comment:
don't think you want statics..

in any case, instead of defining two functions that are the same to use on 4 lines of code, I would use the one line of code directly. Eg

std::transform(measurementType.begin,measurementType.end(),measurementType.begin(), ::tolower);